Letterbook.Api/Controllers/ActivityPub/ActorController.cs

```diff
@@ -1,8 +1,8 @@
 ﻿using System.Reflection;
 using Fedodo.NuGet.ActivityPub.Model.ActorTypes;
 using Fedodo.NuGet.ActivityPub.Model.ActorTypes.SubTypes;
-using Fedodo.NuGet.ActivityPub.Model.CoreTypes;
 using Letterbook.Core;
+using DTO = Letterbook.ActivityPub.Models;
```

Comment:
Does it make sense to put some of this in the Global Usings, some of these models and Activity Pub related content feels like it will be commonly used and establishing a shorthand across all files in the project instead of hoping to copy it between each might help in the long term. Especially with the project already having duplicate names for things like `Actor` - it might cut down on confusion that could cause.

Reply:
That's a good idea. I forget that global using exists, tbh.